install/Knative-with-AKS.md

```diff
@@ -142,46 +142,24 @@ rerun the command to see the current status.
 > Note: Instead of rerunning the command, you can add `--watch` to the above
   command to view the component's status updates in real time. Use CTRL + C to exit watch mode.
 
-## Installing Knative components
+## Installing Knative
 
-You can install the Knative Serving and Build components together, or Build on its own.
-
-### Installing Knative Serving and Build components
+The following commands install all available Knative components. To customize
+your Knative installation, see [Performing a Custom Knative Installation](Knative-custom-install.md).
 
 1. Run the `kubectl apply` command to install Knative and its dependencies:
     ```bash
-    kubectl apply --filename https://github.com/knative/serving/releases/download/v0.2.1/release.yaml
+    kubectl apply --filename https://github.com/knative/serving/releases/download/v0.2.1/release.yaml /
+    --filename https://github.com/knative/eventing/releases/download/v0.2.0/eventing.yaml /
```

Comment:
Global comment: 

The [Eventing install steps](https://github.com/knative/docs/tree/master/eventing#installation) also include the ".../**eventing-sources**/...../release.yaml"

`--filename https://github.com/knative/eventing-sources/releases/download/v0.2.0/release.yaml`